appintro/src/main/res/layout/appintro_intro_layout.xml

```diff
@@ -1,96 +1,100 @@
-
-            <Button
-                android:id="@+id/skip"
-                style="@style/AppIntroButtonStyleCompat"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:layout_gravity="start"
-                android:layout_marginLeft="8dp"
-                android:layout_marginStart="8dp"
-                android:minWidth="100dp"
-                android:labelFor="@string/app_intro_skip_button"
-                android:text="@string/app_intro_skip_button"
-                android:textColor="@android:color/white" />
-
-            <ImageButton
-                android:id="@+id/back"
-                style="@style/AppIntroButtonStyleCompat"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:layout_gravity="start"
-                android:layout_marginLeft="8dp"
-                android:layout_marginStart="8dp"
-                android:minWidth="100dp"
-                android:visibility="invisible"
-                android:contentDescription="@string/app_intro_back_button"
-                app:srcCompat="@drawable/ic_appintro_navigate_before_white" />
-
-            <ImageButton
-                android:id="@+id/next"
-                style="@style/AppIntroButtonStyleCompat"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:layout_gravity="end"
-                android:layout_marginEnd="8dp"
-                android:layout_marginRight="8dp"
-                android:minWidth="100dp"
-                android:contentDescription="@string/app_intro_next_button"
-                app:srcCompat="@drawable/ic_appintro_navigate_next_white" />
-
-            <Button
-                android:id="@+id/done"
-                style="@style/AppIntroButtonStyleCompat"
-                android:layout_width="wrap_content"
-                android:layout_height="match_parent"
-                android:layout_gravity="end"
-                android:layout_marginEnd="8dp"
-                android:layout_marginRight="8dp"
-                android:minWidth="100dp"
-                android:labelFor="@string/app_intro_done_button"
-                android:text="@string/app_intro_done_button"
-                android:textColor="@android:color/white"
-                android:visibility="gone" />
-        </FrameLayout>
-    </LinearLayout>
-</RelativeLayout>
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/background"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/appintro_background_color"
+    android:layoutDirection="ltr"

```

Comment:
This should be removed as this library supports both `rtl` and `rtl`

Reply:
Oh. Okay

Reply:
Done